tests/server/global_config/global_config_test.go

```diff
@@ -235,11 +235,3 @@ func (s *GlobalConfigTestSuite) TestClientWatch(c *C) {
 		}
 	}
 }
-
-func (s *GlobalConfigTestSuite) TestClientWatchTimeout(c *C) {
```

Comment:
Could we delete this gRPC method directly?

Reply:
the test function is not steady, and  the global config is not using.